src/Caching/StackExchangeRedis/src/RedisCache.cs

```diff
@@ -22,20 +22,29 @@ public class RedisCache : IDistributedCache, IDisposable
     // ARGV[3] = relative-expiration (long, in seconds, -1 for none) - Min(absolute-expiration - Now, sliding-expiration)
     // ARGV[4] = data - byte[]
     // this order should not change LUA script depends on it
-    private const string SetScript = (@"
+    private const string LatestSetScript = (@"
```

Comment:
Nit: Consider names other than "Latest..." and "Depreceated...". Maybe "SetScript_HsetBased" and "SetScript_HmsetBased" or something like that?

Reply:
Nothing good comes to mind :(
Imho, nearby comment, explaining why two types of SetScript are used, helps a lot.

Reply:
FWIW I'd name this aligned with the property, e.g. `SetScriptPreExtendedSetCommand` - verbose but super clear.